packages/opentelemetry-web/test/utils.test.ts

```diff
@@ -185,19 +185,23 @@ describe('utils', () => {
         assert.strictEqual(args[1], 123);
       });
     });
-    describe('when entry has time equal to 0', () => {
+    describe('when entry is not numeric', () => {
```

Comment:
I think you should still have unit test when time is 0 - this is basically what you are fixing in this PR right ?

Reply:
Good call; I updated the positive test to check 0, a negative number, and a positive number.